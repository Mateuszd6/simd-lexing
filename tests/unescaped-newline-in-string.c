struct foobar
{
    int a;
    int b;
    itn c;
    int d;
};

int
main(int argc, char** argv)
{
    char const* foo = "And this is a very long string
which has no backslashes and therefore is incorrect";

    // This is a long file so that it does not fit in the first 64-or-so bytes
    return 0;
}
