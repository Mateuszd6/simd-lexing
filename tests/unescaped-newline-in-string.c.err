Here!
./main: ERROR: Unescaped newline in string
