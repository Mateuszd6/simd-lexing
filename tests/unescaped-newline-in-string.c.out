../tests/unescaped-newline-in-string.c:1:1: TOK 0 (L = 6): "struct"
../tests/unescaped-newline-in-string.c:1:8: TOK 0 (L = 6): "foobar"
../tests/unescaped-newline-in-string.c:2:1: TOK 1 (L = 1): "{"
../tests/unescaped-newline-in-string.c:3:5: TOK 0 (L = 3): "int"
../tests/unescaped-newline-in-string.c:3:9: TOK 0 (L = 1): "a"
../tests/unescaped-newline-in-string.c:3:10: TOK 1 (L = 1): ";"
../tests/unescaped-newline-in-string.c:4:5: TOK 0 (L = 3): "int"
../tests/unescaped-newline-in-string.c:4:9: TOK 0 (L = 1): "b"
../tests/unescaped-newline-in-string.c:4:10: TOK 1 (L = 1): ";"
../tests/unescaped-newline-in-string.c:5:5: TOK 0 (L = 3): "itn"
../tests/unescaped-newline-in-string.c:5:9: TOK 0 (L = 1): "c"
../tests/unescaped-newline-in-string.c:5:10: TOK 1 (L = 1): ";"
../tests/unescaped-newline-in-string.c:6:5: TOK 0 (L = 3): "int"
../tests/unescaped-newline-in-string.c:6:9: TOK 0 (L = 1): "d"
../tests/unescaped-newline-in-string.c:6:10: TOK 1 (L = 1): ";"
../tests/unescaped-newline-in-string.c:7:1: TOK 1 (L = 1): "}"
../tests/unescaped-newline-in-string.c:7:2: TOK 1 (L = 1): ";"
../tests/unescaped-newline-in-string.c:9:1: TOK 0 (L = 3): "int"
../tests/unescaped-newline-in-string.c:10:1: TOK 0 (L = 4): "main"
../tests/unescaped-newline-in-string.c:10:5: TOK 1 (L = 1): "("
../tests/unescaped-newline-in-string.c:10:6: TOK 0 (L = 3): "int"
../tests/unescaped-newline-in-string.c:10:10: TOK 0 (L = 4): "argc"
../tests/unescaped-newline-in-string.c:10:14: TOK 1 (L = 1): ","
../tests/unescaped-newline-in-string.c:10:16: TOK 0 (L = 4): "char"
../tests/unescaped-newline-in-string.c:10:20: TOK 1 (L = 1): "*"
../tests/unescaped-newline-in-string.c:10:21: TOK 1 (L = 1): "*"
../tests/unescaped-newline-in-string.c:10:23: TOK 0 (L = 4): "argv"
../tests/unescaped-newline-in-string.c:10:27: TOK 1 (L = 1): ")"
../tests/unescaped-newline-in-string.c:11:1: TOK 1 (L = 1): "{"
../tests/unescaped-newline-in-string.c:12:5: TOK 0 (L = 4): "char"
../tests/unescaped-newline-in-string.c:12:10: TOK 0 (L = 5): "const"
../tests/unescaped-newline-in-string.c:12:15: TOK 1 (L = 1): "*"
../tests/unescaped-newline-in-string.c:12:17: TOK 0 (L = 3): "foo"
../tests/unescaped-newline-in-string.c:12:21: TOK 1 (L = 1): "="
